### ISSUE TITLE
Loss: nan

### ISSUE THREAD

**Issue by cisan86** [ivanwhaf/yolov1-pytorch#4]
hello, I run your code with the default lr=0.02 but get Loss:nan , my batchsize=8, env macos10.15, python3.6,

**Comment by ivanwhaf** [comment]
maybe your learning rate is too big, better lower than 0.01

**Comment by cisan86** [comment]
> 

thx, i try 0.01 and get the right loss. but the output file -- pth file is more than 900mb, is that anything wrong?

**Comment by ivanwhaf** [comment]
probably nothing wrong, see _models/yolov1.py_ for more model-structure details
